lib/src/bottom_sheets/cupertino_bottom_sheet.dart

```diff
@@ -72,6 +72,8 @@ Future<T> showCupertinoModalBottomSheet<T>({
   Color barrierColor,
   bool expand = false,
   AnimationController secondAnimation,
+  Curve routeAnimationCurve,
+  Curve modalAnimationCurve,
```

Comment:
I am not sure about this naming, both are routes so it was not clear for me which one is each.

We could call it animationCurve and, previousRouteAnimationCurve.
What do you think?

Reply:
Yes this is clearer. I will change it.